### PR TITLE
add the api of getCurrentTagIndex and getCurrentTag

### PR DESCRIPTION
**Pull request by neraliu**

### CODE REVIEW

**Comment by yahoocla** [comment]
CLA is valid!

File: src/context-parser.js [hunk 1]
```diff
@@ -341,6 +342,28 @@ FastParser.prototype.getStartTagName = function() {
...
+/**
+ * @function FastParser#getCurrentTag
+ *
+ * @params {integer} The tag Idx
+ *
+ * @returns {string} The current tag name indexed by tag Idx
+ *
+ */
+FastParser.prototype.getCurrentTag = function(tagIdx) {
+    return tagIdx === 0 || tagIdx === 1? this.tags[tagIdx].toLowerCase() : undefined;
```
Comment (maditya) [inline src/context-parser.js]: would tags[tagIdx] ever be undefined? toLowerCase() would fail in such cases

**Comment by maditya** [comment]
also, can we add a couple of unit tests for these functions?

File: src/context-parser.js (hunk 1 above)
Comment (neraliu) [inline src/context-parser.js]: it won't as the constructor will init tags as string array, but we play safe to handle it. :)

**Comment by neraliu** [comment]
added

**Comment by maditya** [comment]
:+1:

**Pull request merged**